target/linux/ath79/dts/ar7161_meraki_mr16.dts

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	compatible = "meraki,mr16", "qca,ar7161";
+	model = "Meraki MR16";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &power_green;
+		led-failsafe = &power_orange;
+		led-running = &power_green;
+		led-upgrade = &power_green;
```

Comment:
Radio Power
Off: MR16 is off
Solid Orange: MR16 is booting or trying to fi nd a path to the internet
Solid Green: MR16 is fully operational and connected to the network
Flashing Orange: Firmware is upgrading
Flashing Green: Error state. May indicate bad gateway or other routing fault 

So, if flashing green indicates error, you shouldn't use that for boot, but use orange for boot and upgrade.

Reply:
Tweaked this. I believe I understood your meaning. I was not able to find the definitions required to cause flashing, however.